ocean/sub_ice_shelf_2D/check_bit_for_bit.py

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/env python
```

Comment:
@xylar I think we can remove this file. If I remember correctly, it pre-dated `check_z_level_bit_for_bit` and we shouldn't need both.

Reply:
Okay, I removed it and cleaned up the commit history.